CI/Bitucket/README.md

```diff
@@ -0,0 +1,73 @@
+# Requirements
+
+* Repository Bitbucket
+
+ ![Create repository](images/create-repository.png)
+ 
+* Application configured to deploy with serverless brefphp
+
+# Step by step
+
+## Bitbucket Configuration
+
+* Go to the "Settings -> Pipelines -> Settings" menu and enable Pipeline:
+![Enable Pipeline](images/enable-pipeline.png)
+ 
+* Access the menu "Settings -> Pipelines -> Deployments"
+    - Select the environment you want to deploy (Staging or Production), in this example I will use Staging
+    - Add the variables
+        * AWS_CREDENTIAL_KEY: Access Key ID
+        * AWS_CREDENTIAL_SECRET: Secret Key
+  
+![Set Environments](images/environment-variables.png)
+
+## Project configuration
+
+* In the root directory of your project create the file `bitbucket-pipelines.yml`:
+```
+pipelines:
+  branches:
+    master: # Whenever there is a push on the master branch
+      - step: # Step 1: Using a docker php image to install composer and download your project dependencies
+          name: Composer install
+          image: php:7.3
+          caches:
+            - composer
+          script:
+            - apt-get update && apt-get install -y unzip
+            - curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+            - composer install --prefer-dist --optimize-autoloader --no-dev
+          artifacts: # artifacts generated during composer install should be moved to the next step
+            - vendor/**
+            - bootstrap/cache/*.php
+      - step:
+          # Step 2: Using the docker node image
+          #  - install serverless
+          #  - inject credentials using previously configured variables
+          #  - deploy
+          name: Deploy Example
+          deployment: staging # WARNING: here you define which deploy variables will be injected into the container according to the previously configured environment.
+          image: node:11.13.0-alpine
+          trigger: manual
+          caches:
+            - node
+          script:
+            - apk add python3
+            - npm install -g serverless
+            - serverless config credentials --stage prod --provider aws --key ${AWS_DEV_LAMBDA_KEY} --secret ${AWS_DEV_LAMBDA_SECRET}
```

Comment:
Setting them like this means that the values are visible in the job output, which could be improved (security-wise).

Instead, if you define the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` variables then the `serverless` command should work out of the box (it looks for those variables specifically).

Reply:
As per the documentation, I need to explicitly use the variable in my yml file.
https://confluence.atlassian.com/bitbucket/set-up-bitbucket-deployments-968683907.html

But you can look at the output of the execution, Bitbucket does not display the value in the output, so there is no security problem with that.

However as recommended, I changed the name of the variables to use the suggested ones.

Thanks.